MonoGame.Framework/MathNumerics.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Xna.Framework
+{
+    public partial struct Vector2
```

Comment:
Put this inside Vector2 instead of making a partial class.

Reply:
The reason because I've put them outside in a separate file is because I am worried about consoles not being able to include System.Numerics.Vectors dependency. If that's the case, a console project can simply remove the "MathNumerics.cs" file from compiling.

I don't how detailed knowledge of the console projects constraints, but if you know that system.numerics.vectors can be used, then you're right and these methods should be moved to their respective classes.

Reply:
If consoles can't support it, we won't be adding it at all, as having an important math API that does not work on consoles defeats the purpose of having it in the first place as 3rd party libraries won't be able to benefit from it.

Should be usable, thats why I'm saying to add it directly.